template/cordova/lib/PluginHandler.js

```diff
@@ -85,12 +85,12 @@ var handlers = {
             var src = obj.src;
             var dest = src;
             var type = obj.type;
+            var targetDir = obj.targetDir || '';
```

Comment:
JavaScript will by default "hoist" variables to the top of functions, which I why I didn't place the variable in the else block.